botocore/args.py

```diff
@@ -781,3 +804,21 @@ def _compute_sigv4a_signing_region_set_config(self, config_kwargs):
                 'sigv4a_signing_region_set'
             )
         config_kwargs['sigv4a_signing_region_set'] = sigv4a_signing_region_set
+
+    def _compute_account_id_endpoint_mode_config(self, config_kwargs):
+        account_id_endpoint_mode = config_kwargs.get(
+            'account_id_endpoint_mode'
```

Comment:
nit - we use `'account_id_endpoint_mode'` 5 times in this function. We can probably create a variable and use that instead:
```py
config_key = 'account_id_endpoint_mode'
...
```